src/scribe_data/utils.py

```diff
@@ -200,27 +222,16 @@ def get_language_words_to_ignore(language: str) -> list[str]:
 
     Returns
     -------
-        The words that should not be included as autosuggestions for the given language as values of a dictionary.
+        list[str]
+            The words that should not be included as autosuggestions for the given
+            language
```

Comment:
No new line (combine with previous line)

Reply:
Thank you! Fixed locally and will send it shortly :)